syntax = "proto3";
// The Fetch service definition.
service Fetch {
  rpc Fetch_CheckOut (Fetch_CheckOutRequest) returns (Fetch_CheckOutReply) {}
  rpc Fetch_CheckIn (Fetch_CheckInRequest) returns (Fetch_CheckInReply) {}
}

message Fetch_CheckOutRequest {
  int32 kit_ID = 1;
  int32 kit_location = 2;
  int32 target_location = 3;
}

message Fetch_CheckInRequest {
  int32 kit_ID = 1;
  int32 kit_location = 2;
  int32 target_location = 3;
}

message Fetch_CheckOutReply {
  bool delivered  = 1;
}

message Fetch_CheckInReply {
  bool returned = 1;
}